### PR TITLE
Fix the issues reported from the SonarCloud

### LINKED ISSUE #62: Fix the issues reported from the SonarCloud
- there are several code smell, bugs and vulnerabilities reported by SonarCloud
   - https://sonarcloud.io/dashboard?id=intellij-common-ui-test-library

### Before:
before

### Now:
Snímek obrazovky 2021-11-01 v 13 10 05

### PR DESCRIPTION
**Pull request by zcervink**
Fixes #62

Signed-off-by: Zbynek Cervinka <zcervink@redhat.com>

### CODE REVIEW

**Comment by zcervink** [comment]
Please review @jrichter1 @odockal

**Comment by zcervink** [comment]
@jrichter1 @odockal Please review.

File: src/main/java/com/redhat/devtools/intellij/commonuitest/UITestRunner.java [hunk 1]
```diff
@@ -168,63 +174,64 @@ public int toInt() {
 
     private static void makeSureAllTermsAndConditionsAreAccepted() {
         String osName = System.getProperty("os.name").toLowerCase();
+        String userHome = "user.home";
 
         if (osName.contains("linux")) {
             step("Copy the 'prefs.xml' file to the appropriate location", () -> {
                 String prefsXmlSourceLocation = "prefs.xml";
-                String prefsXmlDir = System.getProperty("user.home") + "/.java/.userPrefs/jetbrains/_!(!!cg\"p!(}!}@\"j!(k!|w\"w!'8!b!\"p!':!e@==";
+                String prefsXmlDir = System.getProperty(userHome) + "/.java/.userPrefs/jetbrains/_!(!!cg\"p!(}!}@\"j!(k!|w\"w!'8!b!\"p!':!e@==";
```
Comment (odockal) [inline src/main/java/com/redhat/devtools/intellij/commonuitest/UITestRunner.java:182]: Wouldn't it be easier to just define a constant USER_HOME than to call over and over System.getProperty(userHome) ?

Comment (zcervink) [reply src/main/java/com/redhat/devtools/intellij/commonuitest/UITestRunner.java]: Fixed.

**Review by odockal — APPROVED** [review]

**Pull request merged**